### PR TITLE
Transaction statuses (RTD)

### PR DESCRIPTION
**Pull request by neewy**
### Description of the Change

This is the list of all current transaction statuses in Hyperledger Iroha

### Benefits

Explanation of transaction statuses in the docs

### Possible Drawbacks 

None

### CODE REVIEW

File: docs/source/core_concepts/glossary.rst [hunk 1]
```diff
@@ -215,6 +215,39 @@ An ordered set of `commands <#command>`__, which is applied to the ledger atomic
...
+
+Transaction status set
+^^^^^^^^^^^^^^^^^^^^^^
+
+ - NOT_RECEIVED: requested peer does not have this transaction.
+ - STATELESS_VALIDATION_FAILED: the transaction was formed with some fields, not meeting stateless validation constraints. This status is returned to a client, who formed transaction, right after the transaction was sent. It would also return the reason — what rule was violated.
+ - STATELESS_VALIDATION_SUCCESS: the transaction has successfully passed stateless validation. This status is returned to a client, who formed transaction, right after the transaction was sent.
+ - STATEFUL_VALIDATION_FAILED: the transaction has commands, which violate validation rules, checking state of the chain (e.g. asset balance, account permissions, etc.). It would also return the reason — what rule was violated.
+ - STATEFUL_VALIDATION_SUCCESS: the transaction has successfully passed stateful validation.
+ - IN_PROGRESS: the transaction was received, but not yet processed by stateless validator.
```
Comment (lebdron) [inline docs/source/core_concepts/glossary.rst]: No `IN_PROGRESS` here https://github.com/hyperledger/iroha/blob/a91894b4cbae67d5844eaadd44d8128ea1334b66/schema/endpoint.proto#L11-L19

Comment (neewy) [reply docs/source/core_concepts/glossary.rst]: @lebdron じゃあ

Comment (lebdron) [reply docs/source/core_concepts/glossary.rst]: @neewy そうですね

Comment (neewy) [reply docs/source/core_concepts/glossary.rst]: @lebdron I have corrected as you asked, thanks

File: docs/source/core_concepts/glossary.rst [hunk 2]
```diff
@@ -215,6 +215,39 @@ An ordered set of `commands <#command>`__, which is applied to the ledger atomic
 Any nonvalid command within a transaction leads to rejection of the whole
 transaction during the validation process.
 
+Transaction structure
+---------------------
+
+**Payload** stores all transaction fields, except signatures:
+
+    - Time of creation (unix time, in milliseconds)
+    - Account ID of transaction creator (username@domain)
```
Comment (igor-egorov) [inline docs/source/core_concepts/glossary.rst]: Please state here that transaction payload also includes "Quorum" field (indicates required number of signatures).

**Pull request merged**